backend/event/serializers.py

```diff
@@ -0,0 +1,41 @@
+from rest_framework import serializers
+from django.utils import timezone
+from rest_framework.exceptions import ValidationError
+from .models import Event, Invitee, Notification
+from .models import InviteeEventView
+
+class EventSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Event
+        fields = '__all__'
```

Comment:
@sanari2019 It is best not to use the '__all__' fields. It tends to cause bugs by including unwanted fields in the APIs. Happy with this since you are on a time crunch.